.github/workflows/update_ltc.yml

```diff
@@ -0,0 +1,48 @@
+name: Scrape LTC
+
+on:
+  push:
+  schedule:
+    - cron: '30 0 * * *'
+
+defaults:
+  run:
+     working-directory: data-collection-scripts/ltc-scraper
+
+jobs:
+  scrape:
+      runs-on: ubuntu-18.04
+      steps:
+        - uses: actions/checkout@v2
+        - uses: actions/cache@v2
+          with:
+            path: ~/.npm
+            key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+            restore-keys: |
+              ${{ runner.os }}-node-
+        - name: Use Node.js
+          uses: actions/setup-node@v1
+          with:
+            node-version: '14.x'
+        - name: Install dependencies
+          run: npm install
+        # there's an extremely irritating DNS bug somewhere (https://github.com/actions/virtual-environments/issues/798)
```

Comment:
hahah wow

Reply:
It’s not DNS
There is a no way it’s DNS
It was DNS

Do not get me started. It was extraordinarily annoying